src/main/resources/application-custom.yaml

```diff
@@ -25,6 +25,9 @@ spring:
     # database connection pool size
     hikari:
       maximum-pool-size: 25
+      connection-test-query: SELECT 1
```

Comment:
The "connection test query" settings is for legacy drivers and appears to not be recommended anymore.